dashboard/influx-data-gen/influx-data-gen.lua

```diff
@@ -0,0 +1,77 @@
+-- request-gen.lua
+-- usage: 
+-- lua request-gen.lua
+--
+local socket = require("socket")
+local lfs = require("lfs")
+
+local header = 'https,host=ring-0.io,location=rj'
+local flow = 100 -- ~ request/second  (max: 4000)
+
+math.randomseed(os.time())
+
+function read_file_to_table(file_name)
+    local file = io.open(file_name, "r")
+    if not file then
+        print("Error opening the file: " .. file_name)
+        return nil
+    end
+
+    local words_table = {}
+    for line in file:lines() do
+        table.insert(words_table, line)
+    end
+
+    file:close()
+    return words_table
+end
+
+-- traverse lists directory and returns lists table
+function tlists(dir)
+    local tlists = {}
+    local function is_directory(path)
+        return lfs.attributes(path, "mode") == "directory"
+    end
+
+    for file in lfs.dir(dir) do
+        if file ~= "." and file ~= ".." then
+            local subdir_path = dir .. "/" .. file
+            if is_directory(subdir_path) then
+                tlists[file] = {}
+                for subfile in lfs.dir(subdir_path) do
+                    if subfile ~= "." and subfile ~= ".." then
+                        tlists[file][subfile:sub(0,-5)] = read_file_to_table(subdir_path .. "/" .. subfile)
+                    end
+                end
+            end
+        end
+    end
+    return tlists
+end
+
+local lists = tlists("lists")
+
+local FLOW_BASE = 4000
+
+-- generates random pass/warn/block data
+for action, _reason in pairs(lists) do
+  for reason, domains in pairs(_reason) do
+    for _, domain in ipairs(domains) do
+      if(math.random(1,FLOW_BASE/flow) == 1) then
+        print(header .. ' reason="'.. reason ..'",domain="'..domain..'",action="' .. action .. '"' .. " " .. string.sub(math.floor(socket.gettime() * 1000) .. "00".. math.floor(os.clock() * 1000000), 1,19) )
```

Comment:
\n

Reply:
@sav Lua already adds \n in any print() call

Reply:
Sorry, I was not clear, I meant to break the line, it's too long.

Reply:
ok. better.